.github/workflows/rust.yml

```diff
@@ -92,7 +92,7 @@ jobs:
       - name: Install MSRV toolchain
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: 1.53.0
+          toolchain: 1.56.0
```

Comment:
A few lines below `1.53.0` is hardcoded. Perhaps we set an environmental variable and use that in both places within the Actions workflow? i.e.

```yml
env:
  MSRV_VERSION: 1.53.0
```

And
```yml
    - name: Install MSRV toolchain
      uses: actions-rs/toolchain@v1
      with:
        toolchain: ${{ env.MSRV_VERSION }}
        override: true

    - name: check if README matches MSRV defined here
      run: grep '${{ env.MSRV_VERSION }}' README.md
```

Reply:
Yes, good catch!